test/performance/odp_sched_latency.c

```diff
@@ -846,6 +846,63 @@ static int destroy_groups(odp_schedule_group_t group[], int num)
 	return 0;
 }
 
+static int create_queues(test_globals_t *globals)
+{
+	test_args_t *args = &globals->args;
+	int num_group = args->num_group;
+
+	for (int i = 0; i < NUM_PRIOS; i++) {
+		char name[] = "sched_XX_YY";
+		odp_queue_t queue;
+		odp_queue_param_t param;
+		odp_schedule_group_t grp;
+		int prio;
+
+		grp = ODP_SCHED_GROUP_ALL;
+		if (num_group < 0)
+			grp = ODP_SCHED_GROUP_WORKER;
+
+		if (i == HI_PRIO)
+			prio = odp_schedule_max_prio();
+		else
+			prio = odp_schedule_min_prio();
+
+		name[6] = '0' + (prio / 10);
+		name[7] = '0' + prio - (10 * (prio / 10));
+
+		odp_queue_param_init(&param);
```

Comment:
This could be done once outside of loop with `type` and `sync` filled and then fill just the `prio` and `group` during the loop.